course/functions.php

```diff
@@ -96,4 +98,56 @@ function course_register_block_patterns_category() {
 	);
 }
 
-add_action( 'init', 'course_register_block_patterns_category' );
\ No newline at end of file
+add_action( 'init', 'course_register_block_patterns_category' );
+
+/**
+ * Determine the theme variation and save in option.
```

Comment:
The parameters are missing from the DocBlock.

Reply:
Fixed here: https://github.com/Automattic/themes/pull/7034/commits/2ca8a1e0df54aaaad4f425b29c4d09dbe09c5e96